src/local-history/local-history-manager.ts

```diff
@@ -277,4 +274,9 @@ export class LocalHistoryManager {
             fs.statSync(path.join(directoryUri, a)).mtimeMs
         );
     }
+
+    private getHashedFolderPath(uri: string): string {
+        const hashedEditorPath = checksum(uri);
+        return path.join(os.homedir(), '.local-history', hashedEditorPath);
```

Comment:
@Anasshahidd21 to further improve refactoring, can you have `.local-history` as an exportable constant and update anyone that hardcodes the value to instead use it?

Reply:
good idea, I’ll do it.